governance.md

```diff
@@ -0,0 +1,340 @@
+successor, the Steering Council will make a suggestion or suggestions to the
+Main NumFOCUS Board. While the Steering Council and Main NumFOCUS Board will
+work together closely on the BDFL selection process, the Main NUMFOCUS Board
+will make the final decision.
+
+
+Steering Council
+----------------
+
+The Project will have a Steering Council that consists of Project Contributors
+who have produced contributions that are substantial in quality and quantity,
+and sustained over at least one year. The overall role of the Council is to
+ensure, through working with the BDFL and taking input from the Community, the
+long-term well-being of the project, both technically and as a community.
+
+During the everyday project activities, council members participate in all
+discussions, code review and other project activities as peers with all other
+Contributors and the Community. In these everyday activities, Council Members
+do not have any special power or privilege through their membership on the
+Council. However, it is expected that because of the quality and quantity of
+their contributions and their expert knowledge of the Project Software and
+Services that Council Members will provide useful guidance, both technical and
+in terms of project direction, to potentially less experienced contributors.
+
+The Steering Council and its Members play a special role in certain situations.
+In particular, the Council may:
+
+-   Make decisions about the overall scope, vision and direction of the
+    project.
+-   Make decisions about strategic collaborations with other organizations or
+    individuals.
+-   Make decisions about specific technical issues, features, bugs and pull
+    requests. They are the primary mechanism of guiding the code review process
+    and merging pull requests.
+-   Make decisions about the Services that are run by The Project and manage
+    those Services for the benefit of the Project and Community.
+-   Make decisions when regular community discussion doesn’t produce consensus
+    on an issue in a reasonable time frame.
+
+### Council membership
+
+To become eligible for being a Steering Council Member an individual should
+
+-  Be a Project Contributor who has produced contributions that are substantial in
+quality and quantity
+-  Sustain this development consistently over several months
+-  Support the community through code review, answering user questions,
+   triaging bug reports, participating constructively in broader conversations,
+   and being willing to help with documentation and infrastructure.
+-  Demonstrate breadth by supporting the community outside of their particular
+   work interests or sub-project
+-  Be civil in public discourse
+
+Potential Council Members are nominated by existing Council members and voted
+upon by the existing Council after asking if the potential Member is interested
+and willing to serve in that capacity. The Council will be initially formed
```

Comment:
Should we include a percentage threshold required to approve a member?  Is there a way to particular individuals that dissent?

Just to propose a potential solution here, we chose to use [Helios]( https://heliosvoting.org ) for anonymized voting in conda-forge (as some other projects had used it). This works well for us. Not to say that is the way we should go here, but it is one option.

Reply:
It's a good question to ask.  Historically this has been done with lazy-consensus.  Anyone can block, we leave a question open for a while and hope to get a few +1s.  We should probably formalize this though (or select some other approach).

Reply:
Thanks for formalizing it.